browser/css/jsdialogs.css

```diff
@@ -1933,11 +1933,15 @@ input[type='number']:hover::-webkit-outer-spin-button {
 }
 #FormulaDialog .jsdialog.ui-tabpage > .ui-grid {
 	grid-template-rows: 0 auto !important;
+	grid-gap: 0px;
 }
 #FormulaDialog #StructPage {
 	display: grid;
 	grid-template-rows: auto 1fr;
 }
+#FormulaDialog #StructPage .ui-treeview#struct {
+	margin-top: 4px;
+}
```

Comment:
I think we can remove this and instead use the existing declaration `#FormulaDialog #StructPage` and add a `grid-row-gap: 4px;`